src/helpers/utils.js

```diff
@@ -377,8 +377,9 @@ export const getSectionForTranslation = content_type => {
   }
 };
 
-export const noop = () => {
-};
+export const isToday = selectedDate => moment().isSame(moment(selectedDate), 'date');
```

Comment:
Looks like premature refactoring. I would keep this in the ONE place this is being used.
Too early for utils.

Reply:
it looks much more related to utils and not to any code.

Reply:
Don't have a strong opinion about his. Though it will be easier for readers inline.